lldb/source/Plugins/TypeSystem/Clang/TypeSystemClang.h

```diff
@@ -832,7 +832,8 @@ class TypeSystemClang : public TypeSystem {
   // so we catch all names that match a given child name, not just the first.
   size_t
   GetIndexOfChildMemberWithName(lldb::opaque_compiler_type_t type,
-                                const char *name, bool omit_empty_base_classes,
```

Comment:
When we upstream the change to add the exe_ctx, we can also modernize the signature to use StringRef and ArrayRef. And perhaps even return an Optional<>!

Reply:
💯